src/pages/Products/Products.js

```diff
@@ -1,5 +1,40 @@
-import React from 'react';
+/*eslint-disable*/
+import React, { useState } from 'react';
+import './Products.scss';
+import List from './List';
 
 export default function Products() {
-  return <div>Products</div>;
+  const [plus, setPlus] = useState(1);
+  return (
+    <>
+      <div>배너</div>
+      <div className="filter">
+        <ul className="plantsFilter">
+          <li className="plantFilter">식물 모두 보기</li>
+          <li className="plantFilter1">천남성과</li>
+          <li className="plantFilter1">덩굴식물</li>
+          <li className="plantFilter1">양치식물</li>
+          <li className="plantFilter1">소철, 허브</li>
+          <li className="plantFilter1">선인장</li>
+          <li className="plantFilter1">목본류</li>
+        </ul>
+
+        <button className="filterBox">필터</button>
+      </div>
+      <div className="productsMain">
+        <List></List>
```

Comment:
```suggestion
        <List />
```
import 한 컴포넌트는 이렇게 작성하시면 됩니다!